lib/constants.js

```diff
@@ -14,6 +14,14 @@ exports.LOG_ERROR = 'ERROR'
 exports.LOG_WARN = 'WARN'
 exports.LOG_INFO = 'INFO'
 exports.LOG_DEBUG = 'DEBUG'
+exports.LOG_PRIORITIES = [
```

Comment:
I think you missed to add the 'LOG' level itself to constants.js, 
exports.LOG_LOG = 'LOG'

exports.LOG_PRIORITIES = [
      exports.LOG_DISABLE,
      exports.LOG_ERROR,
      exports.LOG_WARN,
      exports.LOG_INFO,
      exports.LOG_DEBUG,
      exports.LOG_DISABLE,
      exports.LOG_LOG
]